app/admin.py

```diff
@@ -108,6 +109,8 @@ def post(self):
                         'deactivation_message_html',
                         'keywords',
                         'launch_status',
+                        'admin_user_email',
```

Comment:
I think that it's better to set 'admin_user_email' and 'unreviewed_notes_email_threshold' per repository since I thought different owners may maintain different repositories. Please let me know the reason why you recommend to set them to globally. I would like to modify as you recommend if I understand the reason. Thanks.